sources/ComicDetailsViewController.swift

```diff
@@ -0,0 +1,190 @@
+////  ComicDetailsViewController.swift
+// Created: 2019-09-21
+//
+
+
+import UIKit
+
+class ComicDetailsViewController: UIViewController {
+
+  var comicId:Int?
+  var comicImage:UIImage?
+  var tags:[String]?
+
+  //Mark: UI Elements
+  let comicImageView: UIImageView = {
+    let imageView = UIImageView(frame: CGRect(x: 0,y: 0,width: 150,height: 150))
+    imageView.contentMode = UIView.ContentMode.scaleAspectFill
+    imageView.clipsToBounds = true
+    return imageView
+  }()
+
```

Comment:
I like how you've organized this code here with the configuration logic in this closure